app/proguard-rules.pro

```diff
@@ -1,6 +1,6 @@
 # Add project specific ProGuard rules here.
 # By default, the flags in this file are appended to flags specified
-# in /Users/ernestwong/Library/Android/sdk/tools/proguard/proguard-android.txt
+# in D:\DavidD\Projects\AndroidStudio\sdk/tools/proguard/proguard-android.txt
```

Comment:
Its commented out so pretty sure its nothing. But i can remove it you want. Usually, i never use proguard so it should be fine either way.